stdlib/public/Concurrency/CMakeLists.txt

```diff
@@ -33,8 +33,7 @@ endif()
 # Frustratingly, in many cases this isn't necessary because the
 # sequence is inlined, but we have some code that's just subtle
 # enough to turn into runtime calls.
-if(SWIFT_HOST_VARIANT STREQUAL "linux" OR
-   SWIFT_HOST_VARIANT STREQUAL "android")
+if(SWIFT_HOST_VARIANT STREQUAL "android")
```

Comment:
This has not been tried on Android, I simply asked that it be added in #38508 since usually it will be needed if it is on linux. If you don't need this on linux after your change, just revert that patch instead, ie remove it for Android too.